internal/tjson/schema.go

```diff
@@ -212,6 +212,25 @@ func (s *Schema) Unmarshal(b []byte) error {
 	return nil
 }
 
+// AddDocumentProperties adds missing $k properties to all the schema's objects (top-level and nested).
+func (s *Schema) AddDocumentProperties() {
```

Comment:
Alternatively, this could be unexproted and could be called from `Unmarshal`. I'm not sure which idea is better.

Reply:
Is it possible to call that method from somewhere else?

Reply:
This is what I do currently - I call it from somewhere else :) 

The question is if we should allow it to be called from somewhere else, or we should add this "missing" JSON schema part right in the place where we unmarshal user-based schema.

To be honest, I'd prefer to add it to `Unmarshal`, but I'm not sure about it.

Reply:
Let's add it to `Unmarshal`. If it would be necessary we always can make exported later on.